### DISCUSSION TITLE
Add Sponsorblock

### DISCUSSION THREAD

**Discussion by AfafAsyraf** [HemantKArya/BloomeeTunes#42] [Ideas]
To skip non music. Kinda suck hearing non music from music video in youtube in this app.

**Comment by HemantKArya** [comment]
Yeah, I admit that it sucks, btw if anyone have any idea about how to skip sponsor block? Currently my focus is on other things in this app so till then if anyone find any idea or api like things to skip then please make sure to discuss it here. 
Thanx 😊

**Reply by kangae-hishiryo** [reply]
[SponsorBlock](https://sponsor.ajay.app/) is actually an API itself:
https://github.com/ajayyy/SponsorBlock
